package.json

```diff
@@ -13,7 +13,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "ci": "CI=true npm run lint && CI=true npm test && npm run build"
```

Comment:
In case you don't have test but configured the linter, you could replace this script by:
```
    "ci": "CI=true npm run lint &&  npm run build"
``
`